python/ray/services.py

```diff
@@ -1043,6 +1061,37 @@ def start_raylet(redis_address,
     return process_info
 
 
+def build_java_worker_command(include_java,
```

Comment:
Maybe this should not take `include_java` and instead only call this method if `include_java` is true.